Lib/test/test_importlib/util.py

```diff
@@ -441,7 +441,7 @@ def contents(self):
         origin='does-not-exist',
         is_package=is_package)
     module.__spec__ = spec
-    module.__loader__ = loader
+    module.__loader__ = module.__loader__ = loader
```

Comment:
Any specific reason to double-assign the same attribute?

Reply:
Nope, good catch!  I thought I needed that in a previous commit, but I don't, so removed!